qoqo_qiskit_devices/qoqo_qiskit_devices/LICENSE_FOR_BINARY_DISTRIBUTION

```diff
@@ -0,0 +1,210 @@
+The source of this software is licensed under the Apache 2.0 license (see below)
+In building the binary distribution other products the software listed in the file DEPENDENCIES are also used and/or linked.
+The licenses an Copyright notices are also listed in the DEPENDENCIES file.
+
+This product in binary form links dynamically against Python  available under the Python license. For full license text see file qoqo_calculator_pyo3/PYTHON_LICENSE
```

Comment:
Minor thing, we should replace `qoqo_calculator_pyo3` with `qoqo_qiskit_devices`